test/unit/testUtil.js

```diff
@@ -154,10 +158,77 @@ define('testUtil', [
         });
     }
 
+    /**
+     * Wait for matching text to be found within the element found by a selector.
+     *
+     * By default, supplying a string to look for, it will use the JS "include()" string
+     * method.
+     *
+     * @param {string} selector Selector for the element within which to look for text
+     * @param {function | string} textOrComparisonFunction Either a string to look for
+     * or function to do the looking; if a string, the JS includes() method is used to
+     * look for it within the element; a function will accept a string to look in, and
+     * will return a boolean indicating whether it was found.
+     * @param {Object} options (optional) several options to control behavior
+     * @param {number} options.timeout (optional) duration, in milliseconds, for which
+     * to look for the string; defaults to 1000ms
+     * @param {number} options.interval (optional) the duration, in milliseconds,
+     * between attempts to find the string; defaults to 100ms
+     * @param {string} options.textName (optional) the a name for the text being looked
+     * for, used in the test failure message; defaults to "text".
```

Comment:
this is a confusing way to describe the keys of the options object (IMO)

Reply:
Isn't that how js doc objects are documented?